pkg/vm/vm_test.go

```diff
@@ -747,9 +747,11 @@ func TestMODMUL(t *testing.T) {
 func TestMODPOW(t *testing.T) {
 	prog := makeProgram(opcode.MODPOW)
 	t.Run("good, positive base", getTestFuncForVM(prog, 1, 3, 4, 5))
-	t.Run("good, negative base", getTestFuncForVM(prog, 2, -3, 5, 5))
+	t.Run("good, negative base", getTestFuncForVM(prog, -3, -3, 5, 5))
+	t.Run("good, negative base", getTestFuncForVM(prog, -1, -1, 3, 3))
 	t.Run("good, positive base, negative mod", getTestFuncForVM(prog, 1, 3, 4, -5))
-	t.Run("good, negative base, negative mod", getTestFuncForVM(prog, 2, -3, 5, -5))
+	t.Run("good, negative base, negative mod", getTestFuncForVM(prog, -3, -3, 5, -5))
+	t.Run("good, negative base, negative mod", getTestFuncForVM(prog, -2, -5, 5, -3))
```

Comment:
This case is not different from existing one in its essence. Why do we need it then?